network-api/networkapi/wagtailpages/templates/buyersguide/product_page.html

```diff
@@ -58,7 +58,7 @@
           {% for cat in product.product_categories.all %}
             {% with category=cat.category %}
               {% localizedroutablepageurl home_page 'category-view' category.slug as cat_url %}
-              <a href="{{cat_url}}" class="tw-no-underline tw-text-gray-60 border tw-border-gray-20 tw-px-2 tw-py-1 tw-font-sans tw-rounded-3xl tw-font-normal tw-text-[12px] tw-leading-[1.3]">
+              <a href="{{cat_url}}" class="category-tags tw-no-underline tw-text-gray-60 border tw-border-gray-20 tw-px-2 tw-py-1 tw-font-sans tw-rounded-3xl tw-font-normal tw-text-[12px] tw-leading-[1.3]">
```

Comment:
Let's rename this to `category-tag`, since it's class-decorating a single element.

Also, in order to distinguish between between the two types of categories, let's add a class that lets us querySelect that distinction, e.g `{% if category.parent == None %}category{% else %}subcategory{% endif %}`.

(And then we'll also want to add the `category-tag sub-category` combination to the subcategory pills that we generate in catalog.html when someone's looking at a specific category)